core/src/main/kotlin/com/malinskiy/marathon/io/FileManager.kt

```diff
@@ -17,6 +17,12 @@ class FileManager(private val output: File) {
         return createFile(directory, filename)
     }
 
+    fun createFile(fileType: FileType, pool: DevicePoolId, device: DeviceInfo, testBatchId: String): File {
```

Comment:
Added new attachments for batch: incomplete batch will save the last screen capture